sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -221,13 +221,8 @@ public override void WriteTo(Stream stream, CancellationToken cancellationToken)
 
             public override bool TryComputeLength(out long length)
             {
-                if (_stream.CanSeek)
```

Comment:
This check seems unnecessary because the constructor throws when `CanSeek` is `false`